bundle/05_spleen_segmentation_lightning.ipynb

```diff
@@ -0,0 +1,1126 @@
+      "Pytorch Ignite version: 0.4.11\n",
+      "ITK version: 5.3.0\n",
+      "Nibabel version: 5.1.0\n",
+      "scikit-image version: 0.21.0\n",
+      "scipy version: 1.11.3\n",
+      "Pillow version: 10.0.1\n",
+      "Tensorboard version: 2.14.1\n",
+      "gdown version: 4.7.1\n",
+      "TorchVision version: 0.15.2+cu117\n",
+      "tqdm version: 4.66.1\n",
+      "lmdb version: 1.4.1\n",
+      "psutil version: 5.9.0\n",
+      "pandas version: 2.1.1\n",
+      "einops version: 0.7.0\n",
+      "transformers version: 4.21.3\n",
+      "mlflow version: 2.7.1\n",
+      "pynrrd version: 1.0.0\n",
+      "clearml version: 1.13.1\n",
+      "\n",
+      "For details about installing the optional dependencies, please visit:\n",
+      "    https://docs.monai.io/en/latest/installation.html#installing-the-recommended-dependencies\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "from monai.config import print_config\n",
+    "\n",
+    "print_config()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c572d8b6-3dca-4487-80ad-928090b3e8ab",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2023-10-06T15:07:34.380130283Z",
+     "start_time": "2023-10-06T15:07:34.330086596Z"
+    }
+   },
+   "source": [
+    "# Spleen Segmentation Lightning Bundle\n",
+    "\n",
+    "In this tutorial we'll describe how to create a bundle for a segmentation network. This will include how to train and apply the network on the command line. Medical  will be used as the dataset with the bundle based off the [Spleen 3D segmentation with MONAI](https://github.com/Project-MONAI/tutorials/blob/main/3d_segmentation/spleen_segmentation_3d_lightning.ipynb) from Spleen segmentation using Task_09 subset from the Medical Segmentation Decathlon.\n",
+    "\n",
+    "This work is licensed under the Creative Commons Attribution-ShareAlike 4.0 International License. To view a copy of this license, visit http://creativecommons.org/licenses/by-sa/4.0/.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1a18d5cd-6338-4b41-87fd-4e119723bfee",
+   "metadata": {},
+   "source": [
+    "Let's start by initialising a bundle directory structure:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "e00b5416-dfab-4043-9293-ec2acdf5842d",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2023-10-16T14:44:10.513242253Z",
+     "start_time": "2023-10-16T14:44:02.546210817Z"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "/usr/bin/tree\n",
+      "SpleenSegLightning\n",
+      "├── configs\n",
+      "│   ├── inference.json\n",
+      "│   └── metadata.json\n",
+      "├── docs\n",
+      "│   └── README.md\n",
+      "├── LICENSE\n",
+      "└── models\n",
+      "\n",
+      "3 directories, 4 files\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%bash\n",
+    "\n",
+    "python -m monai.bundle init_bundle SpleenSegLightning\n",
```

Comment:
After this I would add 
```
rm SpleenSegLightning/configs/inference.json
mkdir SpleenSegLightning/scripts
touch SpleenSegLightning/scripts/__init__.py 
```

so that you don't need to have the empty `writefile` cell below.

Reply:
Thanks!